### PR TITLE
Add governance amount and delegate ID to VeSugar

### PR DESCRIPTION
**Pull request by ethzoomer**

### CODE REVIEW

File: contracts/VeSugar.vy [hunk 1]
```diff
@@ -146,6 +162,12 @@ def _byId(_id: uint256) -> VeNFT:
   amount, expires_at, perma = self.ve.locked(_id)
   last_voted: uint256 = 0
 
+  timepoint: uint256 = convert(self.gov.clock(), uint256)
+  governance_amount: uint256 = self.gov.getVotes(_id, timepoint - 1)
+
+  checkpoint_length: uint48 = self.ve.numCheckpoints(_id)
+  delegate_id: uint256 = self.ve.checkpoints(_id, checkpoint_length - 1).delegatee
```
Comment (stas) [inline contracts/VeSugar.vy:169]: @ethzoomer shouldn't we be using the `self.ve.delegates(_id)` ?

**Review by stas — APPROVED** [review]
Looks good, but not sure if we're doing the most efficient call to get the `delegate_id`.

Let's double check :pray:

**Comment by airtoonricardo** [comment]
> @airtoonricardo If you don't mind pls double check this to confirm if it's the proper delegate implementation for what we discussed earlier

My apologies, I thought the intended use case would be to have the whole checkpoint's information in order to be able to display delegated balances and other information.

Since we are only fetching who the veNFT is delegating to, we should use the `self.ve.delegates(_id)` instead, as Stas has mentioned.

File: contracts/VeSugar.vy [hunk 2]
```diff
@@ -146,6 +155,11 @@ def _byId(_id: uint256) -> VeNFT:
   amount, expires_at, perma = self.ve.locked(_id)
   last_voted: uint256 = 0
 
+  timepoint: uint256 = convert(self.gov.clock(), uint256)
+  governance_amount: uint256 = self.gov.getVotes(_id, timepoint - 1)
```
Comment (stas) [inline contracts/VeSugar.vy:159]: Was taking a closer look at this, do we actually need to call `clock()`, a simple `block.timestamp` should be enough, no?

**Review by stas — APPROVED** [review]
Lets go!

**Pull request merged**